packages/utilities/gel-foundations/index.test.jsx

```diff
@@ -56,6 +56,21 @@ const typographyExpectedExports = {
   GEL_LONG_PRIMER: 'string',
   GEL_BREVIER: 'string',
   GEL_MINION: 'string',
+  getCanon: 'function',
+  getTrafalgar: 'function',
+  getParagon: 'function',
+  getDoublePica: 'function',
+  getGreatPrimer: 'function',
+  getPica: 'function',
+  getLongPrimer: 'function',
+  getBrevier: 'function',
+  getMinion: 'function',
+  getAtlas: 'function',
+  getElephant: 'function',
+  getImperial: 'function',
+  getRoyal: 'function',
+  getFoolscap: 'function',
+  getBodyCopy: 'function',
 };
 
```

Comment:
Please add tests here to check we export all of the scripts e.g. `arabic-pashto`, etc.
This will ensure we don't break any script exports in this module as we maintain this package.